cpp/splinepy/py/py_spline_exporter.hpp

```diff
@@ -23,6 +23,7 @@
 #include <splinepy/py/py_spline.hpp>
 #include <splinepy/utils/print.hpp>
 
+/// @brief
```

Comment:
at this point, I think this file should be called `py_spline_interfaces.hpp` or something instead of exporter.

Reply:
next pr